src/Swarm/Game/State.hs

```diff
@@ -269,11 +271,14 @@ makeLenses ''Notifications
 defaultRobotStepsPerTick :: Int
 defaultRobotStepsPerTick = 100
 
+data Step = WorldTick | RobotStep Ordering
```

Comment:
What is the `Ordering` for?

Reply:
Its to track whether we are running the focused robot (EQ) or robots before/after it.

Once the robots _after_ are finished the game tick is incremented.

Reply:
I see.  Please add a comment! :smile:

Reply:
I added some documentation. :heavy_check_mark: